code/fundamentals.rb

```diff
@@ -7,9 +7,18 @@
 puts message
 
 # 1. Create a variable `my_favorite_animal` and assign it the name of your favorite animal
-
+my_favoite_animal = {
+  name: Joey,
+}
```

Comment:
This is going to cause a problem where `Joey` is not defined. We'd like `my_favorite_animal = "Joey"`